OutOfSchool/OutOfSchool.DataAccess/Models/User.cs

```diff
@@ -9,32 +9,32 @@ public class User : IdentityUser, IKeyedEntity<string>, ISoftDeleted
 {
     public bool IsDeleted { get; set; }
 
+    [Required(ErrorMessage = "LastName is required")]
+    [MaxLength(60)]
+    public string LastName { get; set; }
+
     [DataType(DataType.DateTime)]
     public DateTimeOffset CreatingTime { get; set; }
 
     [DataType(DataType.DateTime)]
     public DateTimeOffset LastLogin { get; set; }
 
-    [Required(ErrorMessage = "LastName is required")]
-    [MaxLength(60)]
-    public string LastName { get; set; }
-
     [MaxLength(60)]
     public string MiddleName { get; set; }
 
     [Required(ErrorMessage = "FirstName is required")]
     [MaxLength(60)]
     public string FirstName { get; set; }
 
-    // If the flag is true, that user can no longer do anything to website.
-    public bool IsBlocked { get; set; } = false;
-
     [MaxLength(50)]
     public string Role { get; set; }
 
     public bool IsRegistered { get; set; }
 
-    // for permissions managing at login and check if user is original provider or its admin
+    // If the flag is true, that user can no longer do anything to website.
+    public bool IsBlocked { get; set; } = false;
```

Comment:
I suggest storing such users like blocked or not confirmed their registration yet in a separate table and use a separate service to handle them.